platform/src/main/java/org/stellar/anchor/platform/config/PropertySep10Config.java

```diff
@@ -137,7 +137,9 @@ void validateConfig(Errors errors) {
             "sep10-web-auth-domain-invalid",
             "The sep10.web_auth_domain does not have valid format.");
       }
-    } else if (homeDomains != null && homeDomains.size() > 1) {
+    } else if (homeDomains != null
+        && !homeDomains.isEmpty()
+        && (homeDomains.size() > 1 || homeDomains.get(0).contains("*"))) {
```

Comment:
Is the wildcard home domain always the first home domain in the list?

Reply:
No this is to make sure `web_auth_domain` must be set if
1.  `homeDomains` has more than one value
or  2.  `homeDomains` has only one value and it is a wildcard domain